Adapters/Vungle/ISVungleAdapter/ISVungleAdapter.m

```diff
@@ -102,74 +79,39 @@ - (NSString *)sdkName {
 
 - (instancetype)initAdapter:(NSString *)name {
     self = [super initAdapter:name];
-    
+
     if (self) {
         if (initCallbackDelegates == nil) {
             initCallbackDelegates = [ConcurrentMutableSet<ISNetworkInitCallbackProtocol> set];
         }
-        
-        // Rewarded video
-        _rewardedVideoPlacementIdToSmashDelegate        = [ConcurrentMutableDictionary dictionary];
-        _rewardedVideoPlacementIdToServerData           = [ConcurrentMutableDictionary dictionary];
-        _rewardedVideoServerDataToSmashDelegate         = [ConcurrentMutableDictionary dictionary];
-        _rewardedVideoPlacementIdsForInitCallbacks      = [ConcurrentMutableSet set];
-        
-        // Interstitial
-        _interstitialPlacementIdToSmashDelegate         = [ConcurrentMutableDictionary dictionary];
-        _interstitialPlacementIdToServerData            = [ConcurrentMutableDictionary dictionary];
-        _interstitialServerDataToSmashDelegate          = [ConcurrentMutableDictionary dictionary];
-        
-        // Banner
-        _bannerPlacementIdToSmashDelegate               = [ConcurrentMutableDictionary dictionary];
-        _bannerPlacementIdToSize                        = [ConcurrentMutableDictionary dictionary];
-        _bannerPlacementIdToViewController              = [ConcurrentMutableDictionary dictionary];
-        _bannerPlacementIdToBannerState                 = [ConcurrentMutableDictionary dictionary];
-        _bannerPlacementIdToServerData                  = [ConcurrentMutableDictionary dictionary];
-        _bannerServerDataToSmashDelegate                = [ConcurrentMutableDictionary dictionary];
     }
-    
+
     return self;
 }
 
 - (void)initSDKWithAppId:(NSString *)appId {
-    
     // add self to the init delegates only in case the initialization has not finished yet
     if ((_initState == INIT_STATE_NONE) || (_initState == INIT_STATE_IN_PROGRESS)) {
         [initCallbackDelegates addObject:self];
```

Comment:
We dont need to manage the Set. u can delete this set completely. @YueVungle

Reply:
@ManojBudumuru-vungle Hi Manoj, we need the initCallbackDelegates which is a global static set used for all ISVungleAdapter instances, please refer to below comment for more details. Thanks a lot. :)
https://github.com/Vungle/levelplay-ios-adapters/pull/1#discussion_r992252070